test/Interop/SwiftToCxx/core/unsigned-return-type-no-zext.cpp

```diff
@@ -4,7 +4,6 @@
 // RUN: %FileCheck %s < %t/ir.ll
 
 // UNSUPPORTED: OS=windows-msvc
-// XFAIL: OS=linux-android, OS=linux-androideabi
```

Comment:
I disabled this last year, 12f20ec75, because it was failing with the clang 14 in NDK 25 at that time, but it builds fine with clang 17 in NDK 26 now.